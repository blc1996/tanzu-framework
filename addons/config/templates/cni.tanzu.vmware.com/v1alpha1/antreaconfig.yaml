#@ load("@ytt:data", "data")
---
apiVersion: cni.tanzu.vmware.com/v1alpha1
kind: AntreaConfig
metadata:
  name: #@ data.values.TKR_VERSION
  namespace: #@ data.values.GLOBAL_NAMESPACE
spec:
  antrea:
    config:
      featureGates:
        EndpointSlice: true
---
apiVersion: cni.tanzu.vmware.com/v1alpha1
kind: AntreaConfig
metadata:
  name: #@ "{}-routable".format(data.values.TKR_VERSION)
  namespace: #@ data.values.GLOBAL_NAMESPACE
spec:
  antrea:
    config:
      trafficEncapMode: "noEncap"
      noSNAT: true
      featureGates:
        AntreaProxy: true
        EndpointSlice: true
