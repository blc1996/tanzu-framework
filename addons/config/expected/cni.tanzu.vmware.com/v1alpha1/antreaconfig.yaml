apiVersion: cni.tanzu.vmware.com/v1alpha1
kind: AntreaConfig
metadata:
  name: v1.23.3---vmware.1-tkg.1
  namespace: tkg-system
spec:
  antrea:
    config:
      featureGates:
        EndpointSlice: true
---
apiVersion: cni.tanzu.vmware.com/v1alpha1
kind: AntreaConfig
metadata:
  name: v1.23.3---vmware.1-tkg.1-routable
  namespace: tkg-system
spec:
  antrea:
    config:
      trafficEncapMode: noEncap
      noSNAT: true
      featureGates:
        AntreaProxy: true
        EndpointSlice: true
